### PR TITLE
WIKI-1255 : HOW-TO Guide template doesn't work properly.

### PR DESCRIPTION
**Pull request by aboughzela**
1- Add Pattern.DOTALL Enables dotall mode. In dotall mode, the expression . matches any character, including a line terminator. By default this expression does not match line terminators. 
2- remove non-printable Unicode character generated by HTMLsanitize

### CODE REVIEW

**Comment by thomasdelhomenie** [comment]
@aboughzela can you explain your changes please ?

File: wiki-service/src/main/java/org/exoplatform/wiki/utils/WikiHTMLSanitizer.java [hunk 1]
```diff
@@ -87,12 +87,15 @@ public static String markupSanitize(String html) throws Exception {
                               + MACRO_END_TAG + "-->");
     }
 
+    // Replace non-printable Unicode character.
+    html = html.replaceAll("\\u200B", "");
```
Comment (boubaker) [inline wiki-service/src/main/java/org/exoplatform/wiki/utils/WikiHTMLSanitizer.java]: Could you plz add this on top of the method.

Comment (aboughzela) [inline wiki-service/src/main/java/org/exoplatform/wiki/utils/WikiHTMLSanitizer.java]: You mean After html = sanitize(html);

**Comment by aboughzela** [comment]
@thomasdelhomenie Description added to PR

**Comment by boubaker** [comment]
please add unit tests.

**Pull request closed without merging**